code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -19,7 +20,7 @@
 		return
 	move_delay = 1
 	if(step(src, direction))
-		spawn(config.walk_speed)
+		spawn(config.walk_speed*move_speed_multiplier)
```

Comment:
although assuing relay_move gets updated every time, you could change this to a

    next_move = worldtime+move_delay
    if(next_move > worldtime) 
        return